### PR TITLE
Upgraded to Nuxt3 RC 13 and added secret to config

### LINKED ISSUE #13: Upgrade to Nuxt 3 stable
Nuxt 3 stable was released, let's:
- [ ] upgrade to it

### PR DESCRIPTION
**Pull request by zoey-kaiser**
Closes #13

### CODE REVIEW

**Comment by BracketJohn** [comment]
have you updated via the nuxi command that they propose in the release notes?

**Comment by zoey-kaiser** [comment]
See discussion about type issue in PR: https://github.com/sidebase/nuxt-session/pull/20 as nuxt-session and nuxt-auth are facing similar issues.

File: src/module.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default defineNuxtModule<ModuleOptions>({
       origin: usedOrigin
     })
 
-    const url = joinURL(options.origin, options.basePath)
+    const url = joinURL(options.origin ? options.origin : 'http://localhost:3000', options.basePath || '')
```
Comment (BracketJohn) [inline src/module.ts:74]: why is this necessary @zoey-kaiser?

both `options.origin` and `options.basePath` are set to their default values above.

**Comment by BracketJohn** [comment]
I've opened an issue with nuxt here: https://github.com/nuxt/framework/issues/8931

Closing this in favor of #22
